azkaban-common/src/main/java/azkaban/flow/FlowUtils.java

```diff
@@ -110,6 +112,20 @@ public static ExecutableFlow createExecutableFlow(final Project project, final F
     return exflow;
   }
 
+  public static ExecutableFlow createExecutableFlow(final Project project, final Flow flow,
+   ExecutorManagerAdapter executorManagerAdapter, Logger logger) {
+    ExecutableFlow exFlow = FlowUtils.createExecutableFlow(project, flow);
+
+    try {
+      executorManagerAdapter.preloadExecutionOptions(exFlow);
+    } catch (ExecutorManagerException e) {
+      if(logger != null) {
+        logger.warn("Fail to preload ExecutableFlow, continue without loading props", e);
+      }
+    }
+    return exFlow;
+  }
+
```

Comment:
Use the new way to create ExecutableFlow so as the `ExecutionOptions` can be there